tests/AppBundle/Controller/CategoryControllerTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Tests\AppBundle\Controller;
+
+use AppBundle\Entity\Category;
+use AppBundle\Entity\Job;
+use Doctrine\ORM\Tools\SchemaTool;
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+
+class CategoryControllerTest extends WebTestCase
+{
+    public function setUp()
+    {
+        $em = $this->getContainer()->get('doctrine')->getManager();
```

Comment:
create tour own TestCase class and move this logic to separate method. It's common case and should be reusable but not written only in CategoryControllerTest class